webapp/app/forms/fields.py

```diff
@@ -19,6 +19,44 @@ def pre_validate(self, form):
         ValidElsterCharacterSet().__call__(form, self)
 
 
+class UnlockCodeWidget(TextInput):
+    """A divided input field with three text input fields, limited to four chars."""
+
+    def __call__(self, field, **kwargs):
+        if 'required' not in kwargs and 'required' in getattr(field, 'flags', []):
+            kwargs['required'] = True
+
+        kwargs['maxlength'] = 4
```

Comment:
Maybe we could set that as a class attribute? Would make this more adaptable for the future?

Reply:
see below